### PR TITLE
Minor fixes in zookeeper event handling.

### PR DESCRIPTION
**Pull request by dpoltavets**

### CODE REVIEW

**Review by niksv — APPROVED** [review]

File: src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/AbstractBolt.java [hunk 1]
```diff
@@ -127,16 +127,30 @@ protected void dispatch(Tuple input) throws Exception {
 
     protected void handleLifeCycleEvent(LifecycleEvent event) {
         if (Signal.START.equals(event.getSignal())) {
-            active = true;
             emit(ZkStreams.ZK.toString(), currentTuple, new Values(event, commandContext));
+            try {
+                activate();
+            } finally {
+                active = true;
```
Comment (surabujin) [inline src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/AbstractBolt.java:134]: I would like to alter `active` value inside `activate()` / `deactivate()` methods, in this case successors can alter behaviour, if required.

File: src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/AbstractBolt.java [hunk 2]
```diff
@@ -127,16 +127,30 @@ protected void dispatch(Tuple input) throws Exception {
...
+            } finally {
+                active = false;
+            }
         } else {
             log.error("Unsupported signal received: {}", event.getSignal());
         }
     }
 
+    protected void activate() {
+    }
```
Comment (surabujin) [inline src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/AbstractBolt.java:149]: add commend "no actions required" or something similar. Now it looks like someone has forgotten to write the method body.

**Review by surabujin — APPROVED** [review]

File: src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/AbstractBolt.java (hunk 1 above)
Comment (dpoltavets) [reply src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/AbstractBolt.java:134]: Good point, but there is no such case yet. For now, we want to always update this flag.

File: src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/AbstractBolt.java (hunk 2 above)
Comment (dpoltavets) [reply src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/AbstractBolt.java:150]: Added.

**Pull request merged**